lib/core/src/Cardano/Wallet.hs

```diff
@@ -278,6 +289,13 @@ data ErrCreateUnsignedTx
     | ErrCreateUnsignedTxFee ErrAdjustForFee
     deriving (Show, Eq)
 
+-- | Errors occuring when estimating transaction fee
+data ErrEstimateTxFee
+    = ErrEstimateTxFeeNoSuchWallet ErrNoSuchWallet
+    | ErrEstimateTxFeeCoinSelection ErrCoinSelection
+    | ErrEstimateTxFeeFee ErrAdjustForFee
```

Comment:
Is this last branch really reachable in `estimateTxFee` ? From the top of my head `ErrAdjustForFee` is only returned by `adjustForFee` which isn't called in the fee estimation. Right?

Reply:
correct!  it is not needed as we do not use `adjustForFees` here